src/unstable/components/Form/widgets/PasswordWidget.tsx

```diff
@@ -39,7 +44,8 @@ export interface PasswordStrengthProps {
 }
 
 const PasswordStrength = ({ password }: PasswordStrengthProps) => {
-	const [strengthScore, setStrengthScore] = React.useState();
+	const [strengthTitle, setStrengthTitle] = React.useState<string>();
+	const [strengthStyle, setStrengthStyle] = React.useState<StrengthStyle>();
```

Comment:
We don't need to keep things in state that can be derived (title can be derived from score, and so can styles).